client/blocks/payment-methods/index.jsx

```diff
@@ -28,6 +28,10 @@ class PaymentMethods extends Component {
 				);
 			}
 
+			if ( method === 'web-payment' ) {
```

Comment:
`web-payment` is under testing on staging, so please don't do that right now :-).

Maybe the best way to handle that is to add `web-payment` in `PAYMENT_METHODS` (in ` client/lib/cart-values/index.js`) if `my-sites/checkout/web-payment` is turned on in the configuration file.

Reply:
Ah, didn't realise it wasn't fully released yet. 

I don't think putting the addition behind a flag would change things unless I disable the feature locally.

This was just to clear up the warning in a semi sane way. If you want the warning to remain I'll close and leave this as is.